### ISSUE TITLE
Possibility of breaking down namespaces by vertical functionality

### ISSUE THREAD

**Issue by rymndhng** [remodoy/clj-postgresql#12]
Is it possible to adopt particular functionality of this project (in particular JSON & Array handling). I'm not interested in adding some of the dependencies into my project (i.e. PostGIS). Is this possible?

**Comment by Atsman** [comment]
I agree, i don't want to have bonecp dependency for example, because i use hicary cp

**Comment by Atsman** [comment]
@rymndhng hi, what we can try to do is split project into subprojects, like in ring project https://github.com/ring-clojure/ring

**Comment by jkjuopperi** [comment]
That makes sense.
